chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/state/RoomUpdatedReducer.kt

```diff
@@ -0,0 +1,16 @@
+package com.pusher.chatkit.rooms.state
+
+import com.pusher.chatkit.state.ChatkitState
+import com.pusher.chatkit.state.UpdatedRoom
+import org.reduxkotlin.reducerForActionType
+
+internal val roomUpdatedReducer =
+    reducerForActionType<ChatkitState, UpdatedRoom> { state, action ->
+        checkNotNull(state.joinedRoomsState)
+
+        state.with(joinedRoomsState = JoinedRoomsState(
+                state.joinedRoomsState.rooms
+                        .plus(action.room.id to action.room),
+                state.joinedRoomsState.unreadCounts
+        ), auxiliaryState = state.auxiliaryState.with(action))
```

Comment:
[likewise](https://github.com/pusher/chatkit-android/pull/192/files#r387086516)

Reply:
Hm, I'm not sure if that link goes to where you think it goes? Or, i don't know what you're asking me to change, sorry :(

Reply:
oh not sure if it's c&p fail of GH but I meant the same formatting as in the other reducer to better utilise vertical space making it more legible (now we have plenty of vertical space in this file ;)

Reply:
👍 understood - have updated the RoomUpdatedReducer to use more vertical spacing.